fineract-provider/src/main/java/org/apache/fineract/portfolio/loanproduct/service/LoanProductReadPlatformServiceImpl.java

```diff
@@ -465,7 +473,7 @@ public LoanProductData mapRow(final ResultSet rs, @SuppressWarnings("unused") fi
                     installmentAmountInMultiplesOf, allowAttributeOverrides, isLinkedToFloatingInterestRates, floatingRateId,
                     floatingRateName, interestRateDifferential, minDifferentialLendingRate, defaultDifferentialLendingRate,
                     maxDifferentialLendingRate, isFloatingInterestRateCalculationAllowed, isVariableIntallmentsAllowed, minimumGap,
-                    maximumGap, syncExpectedWithDisbursementDate, canUseForTopup, isEqualAmortization);
+                    maximumGap, syncExpectedWithDisbursementDate, canUseForTopup, isEqualAmortization, null, this.rates);
```

Comment:
instead of directly passing null, initialize to a variable (variable name similar to the one in called method ) and pass it, for readability. And follow similar approach in all other places

Reply:
Ok